easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '3.0.4'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['2580d66088923a644bc7d3b02efd154b775a3ec3d010426f382bb3be5db9c98b']
+
+builddependencies = [('CMake', '3.9.5')]
+
+dependencies = [
+    ('X11', '20171023'),
+    ('FLTK', '1.3.4'),
+    ('LibTIFF', '4.0.9'),
+    ('tbb', '2018_U5'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread'  -DMPI_INCLUDE_PATH=$EBROOTOPENMPI/include "
+configopts += "-DMPI_C_COMPILER=$EBROOTOPENMPI/bin/mpicc -DMPI_CXX_COMPILER=$EBROOTOPENMPI/bin/mpicxx "
+configopts += "-DCUDA=OFF -DCudaTexture=OFF "
+configopts += "-DALTCPU=ON -DMKLFFT=ON -DFORCE_OWN_TBB=OFF "
```

Comment:
You probably shouldn't have MKLFFT=ON in the foss build.

Reply:
oops... @vanzod, do you want to submit a bugfix PR?

Reply:
Yeah, would be nice to fix that.

Why does it even work though? :-/

Reply:
Fixed in #8043

Reply:
Bugfix done in #8043, thanks @vanzod!

Reply:
It likely detects that there ARE no MKLFFT available and does something else?